### DISCUSSION TITLE
Reference Only Model. I get: RuntimeError: Given groups=1, weight of size [320, 9, 3, 3], expected input[2, 4, 96, 72] to have 9 channels, but got 4 channels instead

### DISCUSSION THREAD

**Discussion by dilectiogames** [Mikubill/sd-webui-controlnet#1256] [Q&A]
anyone knows what is this?

RuntimeError: Given groups=1, weight of size [320, 9, 3, 3], expected input[2, 4, 96, 72] to have 9 channels, but got 4 channels instead

**Comment by dilectiogames** [comment]
nevermind I've selected an inpaint model by mistake.

close call, because automatic1111 didn't wanted to work anymore even if I've have closed controlnet

**Comment by leonviz87** [comment]
same here, i am having some issues, not too sure if anyone can help?
RuntimeError: The size of tensor a (96) must match the size of tensor b (64) at non-singleton dimension 2

this error appeared when i was trying to use the reference model
